releasenotes/notes/0.25/add-pauli-equivalences-74c635ec5c23ee33.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-features:
+features_transpiler:
   - |
     The transpiler's built-in :class:`.EquivalenceLibrary` has been taught the circular Pauli
```

Comment:
I'm not sure the equivalence library is capable of learning :p

Reply:
There's probably more instances of me writing stuff like that - I tend to describe new options as "teaching" the library things haha.